### PR TITLE
Giraffe + text keys

### PR DESCRIPTION
**Pull request by palmfjord**
Like the title says, throw in Apollo and hook it up to the 🦒 and txt 🔑 service.

What's good? SSR + text keys ofc 💥

Thinking i'll replace all texts with text keys in a different request, this is merely the infrastructure for using them.

<strike>I branched off #12 so i figured i'd set that as base branch until its merged</strike>

### CODE REVIEW

File: src/utils/apolloClient.ts [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import { InMemoryCache } from 'apollo-cache-inmemory'
+import { ApolloClient } from 'apollo-client'
+import { BatchHttpLink } from 'apollo-link-batch-http'
+
+export const GIRAFFE_ENDPOINT =
+  typeof window !== 'undefined' &&
+  (window as any).GIRAFFE_ENDPOINT !== undefined
+    ? (window as any).GIRAFFE_ENDPOINT
+    : process.env.GIRAFFE_ENDPOINT || 'https://graphql.dev.hedvigit.com/graphql'
```
Comment (Iamsamwhoami) [inline src/utils/apolloClient.ts]: Maybe we should throw if we can't find the giraffe? would be sad if we by mistake went to the dev environment in prod  😄

Comment (palmfjord) [reply src/utils/apolloClient.ts]: Very good point, that would suck very much. Maybe a check for wether we're in a dev build? I feel like i don't wanna specify these kind of env vars every time i start a dev build of the project

Comment (Iamsamwhoami) [reply src/utils/apolloClient.ts]: @palmfjord sounds good!

Comment (hedvigoscar) [reply src/utils/apolloClient.ts]: We could maybe use this principle in giraffe as well, where the config file defaults to dev-ish values right now in lieu of envvars.

File: src/utils/apolloClient.ts [hunk 2]
```diff
@@ -1,12 +1,27 @@
 import { InMemoryCache } from 'apollo-cache-inmemory'
 import { ApolloClient } from 'apollo-client'
 import { BatchHttpLink } from 'apollo-link-batch-http'
+import { notNullable } from './nullables'
 
-export const GIRAFFE_ENDPOINT =
+let realGiraffeEndpoint: string
```
Comment (Iamsamwhoami) [inline src/utils/apolloClient.ts]: move this into a func and we can avoid the let maybe? 👯‍♂️

Comment (palmfjord) [reply src/utils/apolloClient.ts]: sure 🙌

File: hedvig.config.js [hunk 3]
```diff
@@ -9,5 +9,5 @@ module.exports = {
   port: 8038, // The WDS port
   developmentPublicPath: 'http://0.0.0.0:8038/', // Client public path during development, i.e. "http://0.0.0.0:8081/". Port must match the port directive
   productionPublicPath: undefined, //  Client public path in production, i.e. "/assets/"
-  envVars: ['USE_AUTH', 'AUTH_NAME', 'AUTH_PASS', 'GIRAFFE_ENDPOINT'],
+  envVars: ['USE_AUTH', 'AUTH_NAME', 'AUTH_PASS', 'getGiraffeEndpoint'],
```
Comment (hedvigoscar) [inline hedvig.config.js]: 🤔

Comment (palmfjord) [reply hedvig.config.js]: hahahaha 😂 webstorm is a bit too eager sometimes hehe

**Pull request merged**